content/departments/cloud/technical-docs/v1.1/mi1-1_creation_process.md

```diff
@@ -4,83 +4,53 @@ Creating a new [managed instance](./index.md) involves following the steps below
-1. Open and edit `terraform.tfvars` according to the TODO comments within and commit the result.
+      trial - for customer trial
 
-   > NOTE: ⚠️ Do not set `enable_alerting` to `true` yet as this will cause false alerts to fire until the MI creation process has been completed!
+      production - for paying customer
 
-1. Open and edit `deploy-sourcegraph-managed/$COMPANY/red/docker-compose/docker-compose.override.yaml`, increase `gitserver-0`'s `cpus: 8` if the instance size is larger than "n2-standard-8".
-1. In `deploy-sourcegraph-managed/$COMPANY/project` run `terraform init && terraform apply -var-file=../terraform.tfvars` to create the GCP project
-1. In `deploy-sourcegraph-managed/$COMPANY` run `terraform init && terraform apply`
-1. Check if all is running
+      internal - for internal Sourcegraph usage
 
-   ```bash
-   mg check
-   ```
+  via command line:
 
-1. In the `$COMPANY` directory run `terraform fmt` and commit any changes
+  ```
+  gh workflow run mi_create.yml -f customer=$CUSTOMER -f ce_email=$CE_EMAIL -f customer_email=$CUSTOMER_EMAIL instance_type=[production|trial|internal]
+  ```
 
-1. In the `$COMPANY` GCP project, create [Google Oauth credentials](https://console.cloud.google.com/apis/credentials?project=sourcegraph-managed-$COMPANY) with the following parameters:
+  or via [GithubAction web console](https://github.com/sourcegraph/deploy-sourcegraph-managed/actions/workflows/mi_create.yml)
 
-   - type: Web Application
-   - name: `managed-instance-$COMPANY`
-   - Authorized redirect URIs:
-     - Is the instance **public**, then add **only** `https://$PROJECT_SLUG/.auth/callback`
-     - Is the instance **private**, then add **both** `https://$PROJECT_SLUG/.auth/callback` AND `http://localhost/.auth/callback`
+  Important: GithubAction is `idempotent`, so can be safely re-run multiple times.
 
-1. Create a GCS secret to be used by OIDC authentication
-
-   ```bash
-   mg create-oidc-secret --client-id=<CLIENT_ID_FROM_OAUTH_CREDENTIALS> --client-secret=<CLIENT_SECRET_FROM_OAUTH_CREDENTIALS>
-   ```
-
-1. Initialize the instance and copy the password reset link that is generated by running
-
-   > NOTE: if requested in the new MI request issue, add the CE email in `config.yaml`:
-   >
-   > ```
-   > additionalAdmins:
-   > - CE_NAME@sourcegraph.com
-   > ```
-
-   ```bash
-   mg init-instance $CUSTOMER_ADMIN_EMAIL
-   ```
-
-1. Go back to `terraform.tfvars` and set `enable_alerting` to `true`. Run `terraform apply` and verify that only `google_monitoring_alert_policy.primary` is created.
-1. Commit all changes
+1. PR will be open automatically with the name/branch `$CUSTOMER/create-instance` in [deploy-sourcegraph-managed](https://github.com/sourcegraph/deploy-sourcegraph-managed/pulls) repository.
```

Comment:
Are these bullets done manually, or part of the action? If it's the former, can you add a sentence clarifying that?

Reply:
By GithubAction, I will clarify this.